packages/strapi-plugin-upload/admin/src/components/UploadForm/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { useReducer } from 'react';
+import PropTypes from 'prop-types';
+import { HeaderNavWrapper, ModalBody, ModalForm } from 'strapi-helper-plugin';
+import ModalNav from '../ModalNav';
+import NavLink from '../NavLink';
+import InputFile from '../InputFile';
+import init from './init';
+import reducer, { initialState } from './reducer';
+
+const UploadForm = ({ addFilesToUpload }) => {
+  const [reducerState, dispatch] = useReducer(reducer, initialState, init);
+  const { to } = reducerState.toJS();
+  const links = ['computer', 'url'];
+
+  const handleGoTo = to => {
+    dispatch({
+      type: 'SET_TAB',
+      to,
+    });
+  };
+
+  return (
+    <>
+      <HeaderNavWrapper>
+        <div className="settings-tabs" style={{ left: 30 }}>
```

Comment:
Inline style. This div should be a <SettingsTab> component.

Reply:
Well the className is set in the `HeaderNavWrapper` since I wanted to keep the style from the parent I had no choice. We need to avoid creating components and components when it is not completely needed.